5_9.py

```diff
@@ -0,0 +1,22 @@
+# task_5_9
+"""Для каждого натурального числа в промежутке от m до n вывести все делители,
+кроме единицы и самого числа. m и n вводятся с клавиатуры."""
+
+m = input("Enter m = ")
+n = input("Enter n = ")
+d = {}
+
+try:
+    m, n, = int(m), int(n)
+
+    for number in range(m, n + 1):
+        l = []
+        for i in range(2, n + 1):
```

Comment:
Делитель не будет больше половины от number

```suggestion
        for i in range(2, int(number / 2)):
```